client/src/routes/home/index.tsx

```diff
@@ -1,25 +1,29 @@
 import Leaderboard from "src/components/leaderboard";
 
-import styles from "../../app.css";
+import "../../app.css";
 
 const scores = [
   {
-    name: "Jenny",
+    name: "JENNY",
```

Comment:
Instead of hardcoding the names to uppercase, you can add a `text-transform: uppercase` property in the CSS

Reply:
Yea I agree thats a better solution, could also do `.toUpperCase()` in the tsx file